proposals/stage-2/SUPPORTER_PROGRAM/SPEC.md

```diff
@@ -0,0 +1,58 @@
+# JavaScriptLandia - Phase 1 - Spec
+## October 2020 
+
+### Summary
+JavaScriptLandia is a place where all JavaScript developers, whether if you're all about TypeScript, a React evangelist, or you stick to Node.js, gather to declare their support for the JavaScript ecosystem and OpenJS. 
+
+### Phase 1 Components
+
+**Launch Announcement
+
+This will be a blog post on the OpenJS Site
+Developed in partnership with the content team at the LF 
+We will determine the best date to post with the content team
+
+Design requrements:
+* Image of a country with multiple JS Libraries
```

Comment:
This seems obvious, but we want to make sure this country doesn't look like a country in particular.

Reply:
I think a big square could be a way to ensure that it doesn't resemble any other nation (thinking the big yellow JS square)

Reply:
Agreed.